src/guides/v2.3/config-guide/redis/config-redis.md

```diff
@@ -27,6 +28,41 @@ Installing and configuring the Redis software is beyond the scope of this guide.
 *  [digitalocean](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-redis)
 *  [Redis documentation page](http://redis.io/documentation)
 
+## Set up redis configuration {#config-redis-setup}
+
+Depending on your installation, you can usually find your Redis configuration at /etc/redis/redis.conf or /etc/redis/`port`.conf
+
+To optimize your Redis instance to your needs, you get best results when using a dedicated instance for each sessions, Magento cache and FPC.
+
+For sessions, it's recommended to enable persistence. This can either be done by regular RDB snapshots or by using the AOF persistence logs.
+You can get details of the advantages and disadvantages of RDB and AOF on the [Redis Persistence documentation](https://redis.io/topics/persistence).
+
+RDB (Redis Database Backup) snapshots store the complete database in a dump file after a given time, when a minimum number of keys have changed since the last save.
+This can be configured with the `save` setting inside `redis.conf`.
+
+AOF (Append Only File) stores each write operation sent to Redis in a journal file. This file ẃill be read by Redis on restart to restore the original dataset.
+
+It's possible to enable both RDB and AOF at the same time.
+
+For the cache instance, you should make sure the instance is set up to be large enough to store your whole Magento cache.
+The required size depends on different factors (like number of products and store views) but the required size of the file system cache gives you the order of
+magnitude. Persistence is not required here as the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).
+
+For performance tuning you can also enable these settings for asynchronous deletion. This will not change the behavior of Redis (see also
```

Comment:
Suggested revision of above paragraphs-- also, something missing `(see also..?).   

The size requirement sentence is a bit hard to understand. Not sure what you mean by "file system cache` gives you the order of magnitude.  A specific example might be helpful if it's possible to provide one.


```suggestion
For the cache instance, set up the instance so that it is large enough to store your entire Magento cache.
Size requirements depend on different factors like the number of products and store views, but the size requirement for the the file system cache gives you the order of magnitude. Persistence is not required here as the Magento cache can be restored. See also [Redis cache guide](https://redis.io/topics/lru-cache).

For performance tuning, you can also enable the following settings for asynchronous deletion. These settings do not change the behavior of Redis.
```

Reply:
> Suggested revision of above paragraphs-- also, something missing `(see also..?).
> 
> The size requirement sentence is a bit hard to understand. Not sure what you mean by "file system cache` gives you the order of magnitude. A specific example might be helpful if it's possible to provide one.

I mean that when you have a 20 GB Magento file system cache it won't fit into a 128 MB Redis instance